test/parallel/test-event-emitter-listeners.js

```diff
@@ -24,6 +24,7 @@
 require('../common');
 const assert = require('assert');
 const events = require('events');
+const { kEventListener } = events;
```

Comment:
Why is this required, can it be restructured?

Reply:
Test the wrapped listener has this attr, and the value is the origin listener.